server/tests/api/notes_test.spec.ts

```diff
@@ -0,0 +1,129 @@
+import app from "../../src/app";
+import chai from "chai";
+import chaiHttp from "chai-http";
+import "mocha";
+import NoteCollection from "../../src/models/Notes/NotesCollection";
+
+chai.use(chaiHttp);
+
+const expect = chai.expect;
+
+describe("Notes", () => {
+  afterEach((done) => {
+    NoteCollection.remove({}, (err) => {
+      done();
+    });
+  });
+  /*
+   * Test the /GET route
+   */
+  describe("/GET notes", () => {
+    it("it should GET all the notes", (done) => {
+      chai
```

Comment:
Before testing, can we insert, like an array of 10 notes? We could check better then